robot/Cumulus/resources/RecurringDonationsPageObject.py

```diff
@@ -120,6 +121,9 @@ def edit_recurring_donation_status(self, **kwargs):
         self.selenium.wait_until_element_is_visible(edit_button)
         self.selenium.click_element(locator)
         self.salesforce.wait_until_modal_is_open()
+        self.selenium.reload_page()
+        self.selenium.reload_page()
```

Comment:
Is this needed twice? I am also curios if this will remain in edit mode when reload is called. Let me know if I miss something here.

Reply:
I am not sure, I will let @rjanjanam answer this